pkg/proxy/iptables/proxier.go

```diff
@@ -1244,6 +1234,17 @@ func (proxier *Proxier) syncProxyRules() {
 			writeLine(proxier.natRules, args...)
 		}
 
+		// Next, redirect all src-type=LOCAL -> LB IP to the service chain for externalTrafficPolicy=Local
+		// This prevents traffic originating from the host, but not pods, to be redirected to the service correctly,
```

Comment:
This comment confuses me.  Is it intending to prevent traffic?  It seems to enable traffic?

Reply:
oops! let me fix that